### DISCUSSION TITLE
Physics discussion category?

### DISCUSSION THREAD

**Discussion by elvisish** [godotengine/godot-proposals#5957] [General Discussions]
It could be useful to have one for this, since it doesn't quite fit in any of the other categories and is quite a popular issue.

**Comment by theraot** [comment]
We have 2D and 3D, I suppose those would be graphics then. So:

- 2D Graphics
- 3D Graphics
- 2D Physics
- 3D Physics

That might work. I have also notices the lack of Audio. There might be others worth adding.
